homeassistant/components/tolo/switch.py

```diff
@@ -0,0 +1,51 @@
+"""TOLO Sauna switch controls."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ToloSaunaCoordinatorEntity, ToloSaunaUpdateCoordinator
+from ..switch import SwitchEntity
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up switch controls for TOLO Sauna."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([ToloFanPowerSwitch(coordinator, entry)])
+
+
+class ToloFanPowerSwitch(ToloSaunaCoordinatorEntity, SwitchEntity):
+    """Sauna fan switch control."""
+
+    _attr_icon = "mdi:fan"
+    _attr_name = "Fan"
```

Comment:
If this is a fan, is should be implemented using the `fan` platform instead?

Reply:
Definitely, yes! Completely overlooked that there is a `FanEntity`. Sorry.